index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Innocent's Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    </head>
+    <body class="container">
+        <header class="header">
+            <div class="nav">
+                <a href="#" class="nav-title">Innocent N.</a href="#">
+                <nav href="#">
+                    <img src="./images/menu.png" alt="menu-icon" class ="nav-menu"/>
+                </nav>      
+            </div>
+        </header>
+        <section class="hero">
+            <div class="main">
+                <h1 class="main-header">Hey There.<br>I'm Innocent</h1>
+                <h2 class="sub-header">I'm a Software Developer</h2>
+                <p class="main-paragraph">I can help you build a product,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don't hestitate to contact me.</p>
```

Comment:
[OPTIONAL]Great job so far 👍 . I think it will be better to increase the font-size of this content and give it padding so it can be more clearer.